### DISCUSSION TITLE
Does databend support data and metadata caching?

### DISCUSSION THREAD

**Discussion by rohankrao** [databendlabs/databend#13076] [Q&A]
Does Databend support data and metadata caching for iceberg tables like starrocks?
I am very interested in these features.

**Comment by bohutang** [comment]
I am sure that there is no data caching, cc @Xuanwo

**Reply by rohankrao** [reply]
@BohuTANG Is there any plan to support data and metadata caching of iceberg tables?
Wouldn't it make a big difference in query planning and repeated queries?

**Reply by bohutang** [reply]
@Xuanwo may give some suggestions on this. I think we ca re-use the cache framework which impl for FUSE engine.

**Reply by Xuanwo** [reply]
> Is there any plan to support data and metadata caching of iceberg tables?

Iceberg tables plan to share the same cache framework.
